webpack.base.config.js

```diff
@@ -69,7 +69,7 @@ const baseConfig = {
                         loader: 'esbuild-loader',
                         options: {
                             loader: 'ts',
-                            target: 'es2018',
+                            target: 'es2021',
```

Comment:
Why isn't this driven by tsconfig...?

Reply:
It was a design choice by the webpack team. There was an edge case where you could use the same webpack config for multiple projects and it wouldn't know which of the projects targets it should use. So they just made it explicitly required.